### ISSUE TITLE
[Feature Request] Command line options

### ISSUE THREAD

**Issue by sonicer105** [uintdev/Discord-Cache-Dump#1]
I noticed when I ran the program it required me to press enter to continue and then found both the stable and canary builds I had installed.

Would it be possible to specify which instance to use, possibly with a command line switch?

In additon, could a non-interractive (quite) switch be added to run this program without human interraction?

**Comment by uintdev** [comment]
Hi,
It is possible to use a switch for selecting the build type. I was thinking maybe like `-build=stable`. Does that sound good to you?

**Comment by sonicer105** [comment]
Yes, though you might want to make it `--build stable` and `-b stable` for short to keep it more consistant with other unix like programs.

**Comment by uintdev** [comment]
As `flag` wouldn't allow `--` (only `-`), that means going with `go-cmdline`. An additional requirement, but it can be done.

**Comment by sonicer105** [comment]
Oh, don't worry about the two dashes then if it's going to be a hassle.

**Comment by uintdev** [comment]
Binaries had been released on the release page found at https://github.com/uintdev/Discord-Cache-Dump/releases/tag/v1.2.
The non-interactive flag will not mute or hide the output, but it will not require any user interaction while running. Ultimately, I ditched `go-cmdline` and went with something better (although, the help description will not be dynamic when it comes to the build types available.. not that it matters).
Let me know of what you think of the changes.
